#! /bin/sh /usr/share/dpatch/dpatch-run
## 60_udev_rules_are_autogenerated_notice.dpatch by  <fpeters@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Adds a note to udev rules file to explain that file is generated
## DP: automatically and that local changes will thus be lost on upgrades

@DPATCH@
diff -urNad libgphoto2-2.4.0~/packaging/generic/print-camera-list.c libgphoto2-2.4.0/packaging/generic/print-camera-list.c
--- libgphoto2-2.4.0~/packaging/generic/print-camera-list.c	2007-07-27 02:35:28.000000000 +0200
+++ libgphoto2-2.4.0/packaging/generic/print-camera-list.c	2007-09-28 19:38:45.287336587 +0200
@@ -485,6 +485,7 @@
 		printf ("# udev rules file for libgphoto2 devices (for udev %s version)\n",
 			version_str);
 		print_version_comment(stdout, "# ", "\n", NULL, "#\n");
+		printf ("# this file is autogenerated, local changes will be LOST on upgrades\n");
 		printf (pdata->begin_string);
 	}
 	return 0;
